apps/explorer/app/globals.css

```
@@ -21,7 +21,7 @@ body {
     radial-gradient(circle at 12% 18%, #1c4267 0%, transparent 45%),
     radial-gradient(circle at 84% 8%, #3d2f59 0%, transparent 38%),
     linear-gradient(180deg, #06101a 0%, #08141f 100%);
-  font-family: var(--font-display), sans-serif;
+  font-family: "Space Grotesk", "Inter", "Segoe UI", sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)